src/app/ui/search-track.component.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
Two same components? WTF bro, remove search-track and search-artist, add search-result.component.tsx